libobs/util/windows/window-helpers.h

```diff
@@ -11,6 +11,7 @@ extern "C" {
 enum window_priority {
 	WINDOW_PRIORITY_CLASS,
 	WINDOW_PRIORITY_TITLE,
+	WINDOW_PRIORITY_TITLE_START,
 	WINDOW_PRIORITY_EXE,
```

Comment:
The new entry should be added to the end so that user's data from old version won't be broken.
```suggestion
	WINDOW_PRIORITY_EXE,
	WINDOW_PRIORITY_TITLE_START,
```

Reply:
Why is this conversation marked as resolved?

Reply:
My best guess is because I accepted the change you suggested which automatically marked the conversation as resolved?